### ISSUE TITLE
Moving issue

### ISSUE THREAD

**Issue by Capta1nAlex** [npruehs/ue4-rts#171]
I'm currently studying at 11-th grade and I ain't good enough to understand some parts of manual:
![image](https://user-images.githubusercontent.com/91106292/134105098-3241e4cd-3998-4066-8d4f-972764627c29.png)
I tried to make it with c++, but it didn't work, is there any ways to setup it with blueprints?
I already setted up camera and can select units, but I guess because of my bad C++ understanding I can't end setting PawnAIController and move further ;-( 
I tried to use PawnAIController from your plugin without any changes instead, but it's still not working.
Also I ain't sure about team. Should it be 0, 1 or 255? I can select them all even if they have different index:
![image](https://user-images.githubusercontent.com/91106292/134107674-77718e0d-b466-4296-900d-8f2ed327136a.png)
![image](https://user-images.githubusercontent.com/91106292/134107698-34e0b27f-3632-49a8-ad7c-e108c640b589.png)
![image](https://user-images.githubusercontent.com/91106292/134106606-a35f0c0b-1da1-4e59-9593-7948ae39d932.png)

**Comment by npruehs** [comment]
You're supposed to do that at your blueprint :) 
![image](https://user-images.githubusercontent.com/757321/134475128-92e5002a-70a4-49f1-b402-f5db0ba05026.png)

**Comment by Capta1nAlex** [comment]
![image](https://user-images.githubusercontent.com/91106292/134825539-7daa117b-0bfa-4843-842c-688aed49ee9c.png)
![image](https://user-images.githubusercontent.com/91106292/134825626-8ef9dcfb-d3cc-4846-aeac-0249c7585741.png)
Created BP deriving Added these components to pawn, still not working. 
![image](https://user-images.githubusercontent.com/91106292/134825663-25a48804-55ac-4dc0-a818-ff8df96e9e5d.png)
Navmash also working, I guess... I don't know what's wrong. 
![image](https://user-images.githubusercontent.com/91106292/134825807-452e1dc3-734a-45cc-88a9-e9eac8316e35.png)
I also added gameplay (which wasn't in here previously) and works on 1.25.4. 
![image](https://user-images.githubusercontent.com/91106292/134825856-2c1a3850-e6f1-4cd1-abef-300d299ebd19.png)
Can't even imagine what I missed at this point.

**Comment by npruehs** [comment]
Are your units above ground?

**Comment by Capta1nAlex** [comment]
> Are your units above ground?

No. I created unit from the TopDownCharacter template provided by UE4 and added components from the guide next to it.
![image](https://user-images.githubusercontent.com/91106292/136481466-6c2c4835-6c53-45d4-a9cb-dd3e38b4fd9d.png)

**Comment by npruehs** [comment]
No, I mean - in contrast to "below ground" :)

**Comment by SandorDezso** [comment]
I have the same issue on UE5. The Blackboard says it tries to issue an Attack order.

**Comment by RM6SS62** [comment]
I have the same problem, the order in the blackboard is correct but the location is not, it's not where I click.

Does this even work?
